### DISCUSSION TITLE
Will new bombs be added for version 1.18.2?

### DISCUSSION THREAD

**Discussion by bmd070** [MartinTheDragon/Nuclear-Tech-Mod-Remake#75] [Q&A]
The description of some components says that some of them will be used in other bombs, but they are not.

I would like to know when Ivy Mike, the Tsar Bomba and others will be added, as well as turrets, and will they be added at all?

**Comment by MartinTheDragon** [comment]
Hi!
I don't feel like I can give a date on *when* the remaining nukes will be added, since they aren't the primary focus at the moment. Other than that, everything will get added eventually.
I've been taking a break for a few weeks to focus on things irl, but my current efforts involve bringing NTM to several versions at once with the help of a custom compiler plugin. The project has migrated to [Codeberg.org](https://codeberg.org/MartinTheDragon/Nuclear-Tech-Mod-Remake/) too, so you'll have to look for updates there in the future.
Thanks for playing!

**Reply by doppeltrouble** [reply]
the custom compiler plugin sounds really cool did you make it yourself? because i know some devs who would kill for it.
